docs/data/migration/migration-data-grid-v6/migration-data-grid-v6.md

```diff
@@ -135,6 +135,11 @@ As a result, the following changes have been made:
   };
   ```
 
+- Some feature flags were removed from the `experimentalFeatures` prop. These features are now stable and enabled by default:
+  - [`columnGrouping`](/x/react-data-grid/column-groups/)
+  - [`clipboardPaste`](/x/react-data-grid/clipboard/)
```

Comment:
Nit: This information is duplicated on likes 195 and 309, would removing them or grouping them make it more accessible?

Reply:
Removed the duplicated mentions in favor of a single entry for `experimentalFeatures` changes